packages/gatsby-source-graphql/README.md

```diff
@@ -150,6 +150,49 @@ module.exports = {
 }
 ```
 
+## Composing Apollo Links for production network setup
+
+The plugin provides `createLink` option that you can use to enhance your network stack:
```

Comment:
This intro paragraph could use a bit more context, to help people figure out whether or not this option is something they need.

Possible structure: (Feel free to correct the wording, since I don't totally know how this plugin works)

(You might not actually need the headings, I just added them to highlight the different chunks of information)

---

## Introduction

[Sentence describing the problem (e.g., "Sometimes network requests fail, but that doesn't mean you want your query to fail.")] You can use the plugin's `createLink` option to add an [Apollo Link](https://www.apollographql.com/docs/link/overview/) to your GraphQL operation.

(For more explanation of how Apollo Links work, check out this [Medium article: Productionalizing Apollo Links](https://medium.com/@joanvila/productionizing-apollo-links-4cdc11d278eb).)

## Types of Links

You can create different types of links, depending on the functionality you're trying to achieve:

* `apollo-link-retry` for retrying requests that time out
* [some other lib] for error handling
* [some other lib] for [whatever other thing they're trying to do]

// goal of the list above is to point them toward the right lib if they're trying to do this for the first time

## Parameters

To use the `createLink` option in your `gatsby-config.js`, pass it [explanation of the parameters].

## Example

Here's an example of using the HTTP link with retries (using [apollo-link-retry](https://www.npmjs.com/package/apollo-link-retry)):

[code snippet]

Reply:
Thank you, this is so much better 👍